src/api/deploy/controllers/deploy-service.js

```diff
@@ -79,16 +79,24 @@ const deployServiceController = {
       `Service ${imageName} in ${environment} should be deployed to ${service.zone}`
     )
 
-    await sendSnsDeploymentMessage(
-      deploymentId,
-      payload,
-      service.zone,
-      user,
-      configLatestCommitSha,
-      service.service_code,
-      request
-    )
-    request.logger.info('Deployment sns event sent')
+    // Use file based deployment for infra-dev
+    const shouldDeployByFile = payload.environment === environments.infraDev
```

Comment:
wondering if it would make more sense to either have this as a config setting, either a list of envs or maybe just file-deployments on/off?
We want to deploy it to infra-dev and test deploying to all environments rather than just infra-dev?